### DISCUSSION TITLE
How to play co-op on web browser?

### DISCUSSION THREAD

**Discussion by Tuhin308515** [nzp-team/nzportable#647] [Tutorials]
I tried to follow #400 but it didn't work, there was no 'listening in' line. Could updated instructions be uploaded if possible? Many thanks

**Comment by plooberrest** [comment]
Do everything it says in that but instead of opening solo, open coop and it should work from there. Thats what worked for me.

**Comment by Tuhin308515** [comment]
I tried following #400 but when i'm on step 4, there is no 'Listening on' line in the console. I open a game up in co-op and go to the console and this is all I have. It seems that doing 'sv_public 1' isn't spawning a server.
<img width="727" alt="Screenshot 2024-02-15 110938" src="https://github.com/nzp-team/nzportable/assets/136754178/ffdb3e11-25bc-4dd5-bbcc-05431d3bb4dc">

**Reply by DerpedCrusader** [reply]
#400 specifically says it's outdated, and links to a new method does it work when you follow those steps

**Reply by Tuhin308515** [reply]
My bad i didn't see the new method. I'll have a try and let you know! Thank you! :)

**Reply by Tuhin308515** [reply]
Pretty much, we were trying to play co-op on the WebGL version which is version 3.0 of NZP. But when we installed the Win64 version (which is version 4.6 of NZP), it can work. But what's stopping us is our university's network. So is there anyway to update the WebGL version to latest version i.e. version 4.6 so we can play on the browser rather than having to install it. Many thanks!
